WalletWasabi.Tests/UnitTests/WabiSabi/Integration/Participant.cs

```diff
@@ -18,7 +18,7 @@ namespace WalletWasabi.Tests.UnitTests.WabiSabi.Integration
 {
 	internal class Participant
 	{
-		public Participant(IRPCClient rpc, IBackendHttpClientFactory httpClientFactory)
+		public Participant(IRPCClient rpc, IWasabiHttpClientFactory httpClientFactory)
```

Comment:
Why not rename this param to `wasabiHttpClientFactory`?

Reply:
I get your point but we actually just use this and adding `wasabi` prefix does not help too much to understand it (it's just longer). 

And we don't use typically https://github.com/dotnet/runtime/blob/70065c1b180ebe66dcf68ce03a47e54c23135d79/src/libraries/Microsoft.Extensions.Http/src/IHttpClientFactory.cs#L17 because we have our `IWasabiHttpClientFactory` that switches between Tor & clearnet (.NET's `HttpClient`).

Anyway, is this a convincing argument for / against adding `wasabi` prefix. I don't mind adding it. But I'm ~0 on its value.

Reply:
I don't mind, I just wanted to point that out.